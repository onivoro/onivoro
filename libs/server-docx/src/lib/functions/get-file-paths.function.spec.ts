import { getFilePaths } from './get-file-paths.function';

describe('getFilePaths', () => {
    it('creates a set of unique but conventional strings for inflating/deflating docx files on disk', () => {
        expect(getFilePaths('asdf.docx')).toMatchSnapshot();
    });
});