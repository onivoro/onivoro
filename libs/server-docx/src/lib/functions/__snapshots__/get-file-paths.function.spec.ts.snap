// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getFilePaths creates a set of unique but conventional strings for inflating/deflating docx files on disk 1`] = `
Object {
  "inflated": "asdf/inflated",
  "inputFilePath": "asdf/asdf.docx",
  "outputFileName": "asdf-modified.docx",
  "outputFilePath": "asdf/asdf-modified.docx",
  "root": "asdf",
}
`;
